src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx

```diff
@@ -5,21 +5,24 @@ redirect_from:
   - /sdks/react/integrations/vue-router/
 ---
 
-If you are using Vue Router, you should use our provided integration to get better, parameterized transaction names. Here is a full example of how to use it:
+If you are using Vue Router, you should use our provided routing instrumentation to get better, parameterized transaction names.
```

Comment:
```suggestion
If you're using Vue Router, you should use our provided routing instrumentation to get better, parameterized transaction names.
```

Reply:
There'll come a day when I remember this but the last 12 years of uni/school telling me not to use these abbreviations still have their influence on me. So thanks for always correcting these ;)